src/fontra/views/fontinfo/panel-sources.js

```diff
@@ -21,7 +27,30 @@ import "/web-components/add-remove-buttons.js";
 import "/web-components/designspace-location.js";
 import { dialogSetup, message } from "/web-components/modal-dialog.js";
 
-const cardsInfos = {};
+let selectedSourceIdentifier = undefined;
+
+addStyleSheet(`
+.font-sources-container {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  height: calc(100vh - var(--top-bar-height));
```

Comment:
Why is this needed? The total page already has a calculation for the top bar. A panel like this should not have to worry.

Reply:
This is required to make the two elements (source names, source content) scrollable separately. Without it the the whole container scrolls, but not each element separately.

Reply:
There must be something wrong elsewhere then, because the container for this container should already have the correct size.

Reply:
We fixed this together. I am closing this as resolved.